src/tests/ftest/control/dmg_nvme_scan_test.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from __future__ import print_function
+
+import os
+
+from dmg_utils import DmgCommand
+from server_utils import ServerCommand
+from apricot import TestWithServers
+from avocado.utils import process
+
+class DmgNvmeScanTest(TestWithServers):
+    """Test Class Description:
+    Simple test to verify the scan function of the dmg tool.
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a DmgNvmeScanTest object."""
+        super(DmgNvmeScanTest, self).__init__(*args, **kwargs)
+        self.setup_start_servers = False
```

Comment:
What's the reason for not just allowing `TestWithServers().setUp()` to start the servers given that you start them right at the beginning of this test?

Reply:
There is no explicit reason except to use the ServerCommand.start() implementation of running the servers. Either method could be used... when starting the servers with TestWithServers().setUp(), the server_utils.run_server() function is used and the plan is to eventually use ServerCommand.start() to replace run_server() entirely.

Reply:
That sounds like it should be in a separate PR as it's own feature rather than being lumped on top of this one.

In it's own PR, it would replace the `server_utils.run_server()` functionality in _apricot_'s `setUp()` wouldn't it?